src/Ledger/Conway/Conformance/Equivalence.agda

```diff
@@ -148,7 +149,7 @@ instance
       ledger' : Γ C.⊢ (getCertDeps* cdeposits ⊢conv s) ⇀⦇ tx ,LEDGER⦈ C.⟦ utxoStC' , govSt' , certStateC' ⟧ˡ
       ledger' = C.LEDGER-V⋯ refl utxow' certs' gov'
       utxoEq  : utxoStC' ≡ utxoSt'
-      utxoEq  = cong (λ • → L.⟦ _ , _ , • , _ ⟧ᵘ)
+      utxoEq  = cong (λ • → ⟦ _ , _ , • , _ ⟧)
```

Comment:
I'm surprised you can remove both the superscript and the `L.` here. 🤔  I suppose I should try to understand the metaprogramming you did to see why this works.

Reply:
This is about an instance of `To` for `RewardsUpdate` being in scope.